Makefile

```
@@ -3,7 +3,12 @@ REPO_ROOT                                      := $(shell dirname $(realpath $(l
 GITHUBORG                                      ?= open-component-model
 OCMREPO                                        ?= ghcr.io/$(GITHUBORG)/ocm
 VERSION                                        := $(shell go run api/version/generate/release_generate.go print-rc-version $(CANDIDATE))
+COMMIT                                         = $(shell git rev-parse HEAD)
```

```suggestion
COMMIT                                         = $(shell git rev-parse --verify HEAD)
```
not sure if it's required or not...

---

I think its good practice because it makes sure HEAD is not corrupted. so I added it everywhere